invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/AccessField/components/ProtectionButtons.js

```diff
@@ -110,6 +110,6 @@ export class ProtectionButtons extends Component {
 }
 
 ProtectionButtons.propTypes = {
-  canRestrictRecord: PropTypes.bool.isRequired,
+  canRestrictRecord: PropTypes.bool,
```

Comment:
It Looks like it's still missing `defaultProps` declaration for the `canRestrictRecord` prop?

Reply:
didn't saw that on the browser. will check the next time the tests too. beside that:
what should be the default value for it?

Reply:
The default value appears to be true as per the reference [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/a4fbec8dbde537244a58d905495fb6919029abaa/invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/AccessField/components/MetadataAccess.js#L33C1-L44C1). Feel free to double-check!